### ISSUE TITLE
Aqara temp sensors randomly reporting wrong values

### ISSUE THREAD

**Issue by shaarkys** [TedTolboom/com.xiaomi-mi-zigbee#217]
As mentioned on community forum. visible as well on screenshot.

![SmartSelect_20200930-205551_Samsung Internet](https://user-images.githubusercontent.com/14859895/94727758-7be03600-035f-11eb-87c0-30a6ba5dad37.jpg)

Thank you.

**Comment by shaarkys** [comment]
Note - gaps are caused by turned off Homey

**Comment by robiebab** [comment]
Hi Ted, first off all thanks for the work! second, im trying to add a new aquara temperature sensor to homey without luck. after selecting the device out of the list is tells me to press 3 seconds on the button, i did that and also longer without and luck. i also check the zigbee network, the device is not included, any ideas?

**Comment by TedTolboom** [comment]
@shaarkys which data is it actually reporting; you mentioned -100 degrees for temperature?

@robiebab please contribute to the issue as described by @shaarkys in this issue or create a new issue, to discuss your issue (doesn't seem correlated to this one) there

**Comment by shaarkys** [comment]
Sorry @TedTolboom , it.was + 100 degrees of Celsius (because I have notification enabled which shows Temperature-xx that's why I mixed it, instead of negative it was positive)

**Comment by TedTolboom** [comment]
ehm.. 100% (for humidity) and -100 degrees for temperature.. al least that's consistent with your graphs...
Could you otherwise create a crash log within the app and share the details, I can then read the actual reported data.

Settings > Apps > Aqara app > Settings > Create Diagnostics Report

And share the ID fo the report in this issue

**Comment by shaarkys** [comment]
You are right, sorry, too tired today - 100% humidity and +100 degrees of Celsius. You mean to create crash report when/if happens again, right. Because so far it happened twice only.

**Comment by TedTolboom** [comment]
No, just hit the crash report now... it will show some device logs

**Comment by shaarkys** [comment]
Here you go Ted : 54124f91-3ad8-4e0c-a339-847facc8edbe

(I was still on GitHub version, so I wait a little bit after reinstall)

**Comment by TedTolboom** [comment]
Nah... you just restarted before the crashlog... 

```
---- stdout ----
2020-10-01 20:11:20 [log] [XiaomiZigbee] Xiaomi Zigbee app is running...
2020-10-01 20:11:29 [log] [ManagerDrivers] [cube] [0] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sens] [0] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_magnet] [0] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_magnet.aq2] [0] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_magnet.aq2] [1] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_magnet.aq2] [2] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_motion] [0] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_motion] [1] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_motion] [2] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_motion] [3] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_motion] [4] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_motion] [5] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_motion.aq2] [0] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_motion.aq2] [1] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_motion.aq2] [2] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_switch] [0] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_switch] [1] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_switch.aq2] [0] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_switch.aq2] [1] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [sensor_switch.aq3] [0] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [vibration.aq1] [0] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [vibration.aq1] [1] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [vibration.aq1] [2] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:29 [log] [ManagerDrivers] [vibration.aq1] [3] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:31 [log] [ManagerDrivers] [weather] [0] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:31 [log] [ManagerDrivers] [weather] [1] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:31 [log] [ManagerDrivers] [weather] [2] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:31 [log] [ManagerDrivers] [weather] [3] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:31 [log] [ManagerDrivers] [weather] [4] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:11:31 [log] [ManagerDrivers] [weather] [5] ZigBeeDevice has been initialize { firstInit: false, isSubDevice: false }
2020-10-01 20:13:15 [log] [ManagerDrivers] [sensor_magnet.aq2] [2] lifeline attribute report { batteryVoltage: 2975 }
2020-10-01 20:14:07 [log] [ManagerDrivers] [sensor_motion.aq2] [2] lifeline attribute report { batteryVoltage: 3025 }
2020-10-01 20:14:56 [log] [ManagerDrivers] [weather] [0] lifeline attribute report { batteryVoltage: 2915, state: 1520, state1: 6977, pressure: 97036 }
2020-10-01 20:14:56 [log] [ManagerDrivers] [weather] [0] measure_temperature | msTemperatureMeasurement - measuredValue (temperature): 15.2 + temperature offset 0
2020-10-01 20:14:56 [log] [ManagerDrivers] [weather] [0] measure_humidity | msRelativeHumidity - measuredValue (humidity): 69.8 + humidity offset 0
2020-10-01 20:14:56 [log] [ManagerDrivers] [weather] [0] measure_pressure | msPressureMeasurement - measuredValue (pressure): 970 + pressure offset 0
2020-10-01 20:15:31 [log] [ManagerDrivers] [vibration.aq1] [2] closuresDoorLock - 1285 (vibration): 458752 7
2020-10-01 20:17:38 [log] [ManagerDrivers] [sensor_motion.aq2] [1] lifeline attribute report { batteryVoltage: 3005 }
2020-10-01 20:20:25 [log] [ManagerDrivers] [vibration.aq1] [3] lifeline attribute report { batteryVoltage: 3005 }
```

**Comment by shaarkys** [comment]
I had to install Homey store test version as I was on GitHub version still, otherwise there is no Diagnostic report option... So I will post it on next reoccurrence then...

**Comment by TedTolboom** [comment]
Thanks

**Comment by shaarkys** [comment]
New report Ted : 87abad92-4484-4050-8ed5-0a5ab8f23d6d

**Comment by shaarkys** [comment]
What is strange this time, values seems to be stuck now...maybe it's HW malfunction?

![SmartSelect_20201004-080003_Homey.jpg](https://user-images.githubusercontent.com/14859895/95008185-d7d4e400-0617-11eb-97e5-09f9c5f69047.jpg)

![SmartSelect_20201004-080929_Samsung Internet.jpg](https://user-images.githubusercontent.com/14859895/95008311-f9829b00-0618-11eb-8c1f-05fb2fc1adfe.jpg)

**Comment by shaarkys** [comment]
@TedTolboom new report if it helps, sensor again reports correct values : 50554452-1dea-4943-af64-9b4e45da840e

![obrazek](https://user-images.githubusercontent.com/14859895/95205473-b0cc0d00-07e5-11eb-88bd-7befaae05e01.png)

**Comment by shaarkys** [comment]
OK, so I was thinking if this is not caused by faulty battery (> 1 year) and look how the graph looks like since I exchanged battery (and unfortunately re-powered device in parallel). Anyway, maybe it shall remain as it is, because it is indicating potential issues.

![obrazek](https://user-images.githubusercontent.com/14859895/95506919-ee30c600-09b0-11eb-925f-7815bf1b4b49.png)

**Comment by aivus** [comment]
I'm experiencing the same here.

I've prepared app diagnostic report. It's ID: 1316bf45-c7d1-4abb-8a94-247fbe42c556

![image](https://user-images.githubusercontent.com/1021703/103304815-43a9cb80-4a12-11eb-9fa5-c1ee425e91e7.png)

**Comment by HenkRenting** [comment]
Seems I do have a Github account after all... :)
(see my post in the community topic)

i am experiencing the same problem.
This my diagnostic report: 9341abe6-db5c-4b1d-b42a-1fb7fb9b9c03
Hope you don’t mind pointing here to my post at the community, since I do not have the screenshots anymore (simply copy-pasted them). Screenshots can be seen here:
https://community.athom.com/t/aqara-xiaomi-smart-home-zigbee-app-v0-8-0-test-v1-2-3/156/2826

Note: Notices peaks and drops when on v5rc59. Just updates to rc60 (made report after). Haven’t noticed the problem on rc60, bit it is on only a few minutes.

**Comment by shaarkys** [comment]
I don't see this behavior anymore, thank you

**Comment by HenkRenting** [comment]
Me neither. This was a post from 2 years ago after all...

**Comment by shaarkys** [comment]
Yep, just clean-up... ;-)
